configure.ac

```diff
@@ -298,7 +298,7 @@ PCS_CHECK_PYMOD([pyOpenSSL])
 # those are kind of problematic.
 # use them all from the BaseOS or embedded them all as necessary (--enable-local-build)
 PCS_CHECK_PYMOD([dacite], [], [yes])
-PCS_CHECK_PYMOD([tornado], [>= 6.0.0], [yes])
+PCS_CHECK_PYMOD([tornado], [>= 6.1], [yes])
```

Comment:
@tomjelinek it fails CI for latest autotools and stuff. You want that to be 6.1.0 (see https://ci.kronosnet.org/job/pcs-build-all-voting/pcs-build-all-voting=ubuntu-20-10-x86-64/lastBuild/console) and there is an unrelated failure here:

https://ci.kronosnet.org/job/pcs-build-all-voting/pcs-build-all-voting=rhel83z-s390x/lastBuild/console

Reply:
Apology, I misread the log. Ubuntu-20-10 ships only 6.0.4. If 6.1 is a requirement now, then we need to drop support for Ubuntu.